flake.nix

```diff
@@ -54,7 +54,7 @@
       in {
         # development shell
         devShells.default = pkgs.mkShell {
-          buildInputs = [ pkgs.openjdk21 pkgs.clojure pkgs.babashka depsCache ] ++ (basicToolsFn pkgs);
+          buildInputs = [ pkgs.openjdk21 pkgs.clojure pkgs.babashka pkgs.nodejs_20 depsCache ] ++ (basicToolsFn pkgs);
```

Comment:
Notably, there are also `nodePackages` available in nixpkgs that we currently don't use (preferring to make you `npm install`). But in theory we could add those packages here too. This change makes sense to me as is though.

Reply:
Ahh, I wasn't sure. I see it in nixpkgs now.

I feel like incompleteness is always such a problem. Like, if there's any package not available in nix, you have to fall back to package.json anyway, and then... what? Merges and conflicts? Human sacrifice, dogs and cats living together, mass hysteria!